javascript/javascript3/week1/week1.json

```diff
@@ -0,0 +1,20 @@
+{
+    "music1": {
+        "artist": "Brett Kissel",
+        "album" :"Now or Never",
+        "genre":  "Country",
+        "label" : "Warner music Canada"
+    },
+     "music2" : { 
+        "artist": "Stabbling Westward",
+        "album" :"Dead and Gone",
+        "genre":  "Industrial Rock",
+        "label" : "Drug Store Records"  
+    },
+    "music3" : {
+        "artist": "Poppy",
+        "album" :"I disagree",
+        "genre":  ["Heavy Metal","Pop","Elcro Pop"],
+        "label" : "Sumerian"  
+    }
+}
```

Comment:
Consider using array and if you need a name simply add a new name/value pairs to it.

[
    {
       "id": 1,
        "artist": "Brett Kissel",
        "album" :"Now or Never",
        "genre":  "Country",
        "label" : "Warner music Canada"
    },
     { 
       "id": 2,
        "artist": "Stabbling Westward",
        "album" :"Dead and Gone",
        "genre":  "Industrial Rock",
        "label" : "Drug Store Records"  
    },
     {
       "id": 3,
        "artist": "Poppy",
        "album" :"I disagree",
        "genre":  ["Heavy Metal","Pop","Elcro Pop"],
        "label" : "Sumerian"  
    }
]